pkg/api/converterfromplugin.go

```diff
@@ -11,7 +11,8 @@ func ConvertFromPlugin(in *plugin.Config, old *Config) *Config {
 	if old != nil {
 		cs = old.DeepCopy()
 	}
-	cs.ClusterVersion = in.ClusterVersion
+	cs.PluginVersion = in.PluginVersion
+	cs.RPVersion = in.RPVersion
```

Comment:
Think RPVersion should not appear in converter{to,from}plugin

Reply:
We potentially want to have it saved inside config internal API. But we dont want it to appear inside external API. Only Admin API, so we could query and check it. At the same time adminAPI should not be able to modify it. No?

Reply:
This is why it does not exist in convertFromAdmin function.

Reply:
we're talking about different things.
admin API converters: should exist in both directions - via convertfromadmin is how SRE will trigger a cluster update
plugin API converter: not convinced it should be there, check what MSFT wants.